lib/new_relic/version.rb

```diff
@@ -7,7 +7,7 @@
 module NewRelic
   module VERSION # :nodoc:
     MAJOR = 8
-    MINOR = 10
+    MINOR = 11
```

Comment:
Since we only have bugfixes, should this be a patch version change? (8.10.1?)

https://semver.org/

Reply:
That's a good point! It's less common that we have a release like this that's just a couple bugfixes and nothing new, so I just autopilot put 8.11, but I can update that!

Reply:
changed in fbc8166898507d858d25b5f094775d695c6dbe77